src/internal/master_actor.cc

```diff
@@ -460,7 +460,7 @@ caf::behavior master_state::make_behavior() {
   BROKER_TRACE(BROKER_ARG(id) << BROKER_ARG(core) << BROKER_ARG(store_name));
   // Setup.
   self->monitor(core);
-  self->set_down_handler([this](const caf::down_msg& msg) {
+  self->set_down_handler([this](const caf::down_msg& msg) { //
```

Comment:
I've added this manually to force clang-format to break it here. Otherwise, clang-format breaks before the lambda and adds a weird extra indentation level that makes it harder to read. There are actually a couple places I do this in the code base.

Reply:
Ah, right on.